service/src/main/java/uk/gov/hmcts/reform/fpl/model/Direction.java

```diff
@@ -20,6 +22,8 @@
     private String directionText;
     private final String status;
     private DirectionAssignee assignee;
+    private ParentsAndRespondentsDirectionAssignee parentsAndRespondentsAssignee;
```

Comment:
In healthy environment it makes more sense to have it in more specialised class. Here `Direction` class became one big bag of properties not helping to understand which one are needed and when :(